web/src/lib/enums/httpEnum.js

```diff
@@ -1,3 +1,8 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+
```

Comment:
This file is from third-party component which under the MIT license, why do you add apache license here?
When we change the ts file to js file, which license should we choose? What do you think? @ch3yne

Reply:
Hi Xia, I have the same question. Although the original file has MIT license, but it cannot pass the test. It will cause such error:
> Task :rat
Files with unapproved licenses:
 - /home/lance/gravitino/web/src/lib/enums/httpEnum.js
> Task :rat FAILED

Tha's why I add a license later. And you are right, it is not suitable to add apache license. So which license shall we choose?

Reply:
We cannot change headers on 3rd party files. If a file has no header leave it as it is and add it to the rat exclusions. The file will also be needed to be mentioned in LICENSE and LICENSE.bin and its license text added.

Reply:
@LanceHsun You should replace ts files to js files of the exclusions list at the line 463 of web/build.gradle.kts

Reply:
Thank you for reminding me! Done!